### PR TITLE
Finished updating step 1 and 2

### PR DESCRIPTION
**Pull request by mostaphaRoudsari**
Here are the list of changes:
1. Renamed folders and added links to main README 
2. Updated README for creating k8s cluster
I added more explanations and how it will look like when you run the commands in shell. I also added the link for installing kubectl. The only major change that I made was removing the part for saving the kubectl credentials.  As far as I can see if you doctl saves the config file when the cluster is created. Also the command is missing cluster. I already reported this: digitalocean/doctl#395
3.  updated management dashboard README: Mostly added more explanation to each command. Also the link to yml file on GitHub was broken which I replaced it with the local yml file.

@AntoineDao I didn't get part: "(if using RBAC - on by default on new installs with kops / kubeadm)". Is there a link to read more about this?

### CODE REVIEW

File: 02-management-dashboard/README.md [hunk 1]
```diff
@@ -1,37 +1,58 @@
...
-kubectl proxy
+$ kubectl proxy
+Starting to serve on 127.0.0.1:8001
 ```
 
 ## Login to dashboard
 Go to http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/#!/login
 
-Choose login token and enter the login token from the previous step
+Choose login token and enter the login token from the previous step. Congratulations! :)
```
Comment (AntoineDao) [inline 02-management-dashboard/README.md]: Very important to congratulate folk on their journey 😄

File: 01-digital-ocean-cluster/README.md [hunk 2]
```diff
@@ -1,29 +1,75 @@
...
-> kubectl get nodes
-which should return something like:
 ```console
+~/.kube$ kubectl get nodes
 NAME                   STATUS   ROLES    AGE     VERSION
-fervent-poitras-836n   Ready    <none>   4m13s   v1.12.1
-fervent-poitras-83a9   Ready    <none>   4m14s   v1.12.1
-```
\ No newline at end of file
+fervent-poitras-836n   Ready    <none>   4m13s   v1.13.1
```
Comment (AntoineDao) [inline 01-digital-ocean-cluster/README.md]: This version change is for the Kubernetes server running on Digital ocean. It's likely to change as new deployments get set up. As the `kubectl get nodes` is for demo purposes it's not essential to change I'd say but good to have handy 👍

File: README.md [hunk 3]
```diff
@@ -3,28 +3,25 @@ This mono repo contains code to provision, manage and deploy to a kubernetes clu
...
+2. [Deploy a kubernetes dashboard pod/service on the cluster to facilitate debugging](./02-management-dashboard/README.md).
+3. [Deploy `istio` with mTLS enabled](./03-istio/README.md).
+4. [Set up an externel load balancer to provide a domain name to access your cluster's services](./04-externalDNS/README.md).
+5. [Set up a Postgres Database with a mounted volume which will presist even if the entire cluster is shut down](./05-postgres-database/README.md).
 ...
-
-9. `take a break`!
-
+6. **take a break!**
+7. Deploy the rest of the microservices as needed!
```
Comment (AntoineDao) [inline README.md]: Yes! Sorry my bad I forgot to remove the microservice bit.

**Comment by AntoineDao** [comment]
Regarding RBAC (Role Based Access Control) it is essentially a way to create users/roles inside your kubernetes cluster with certain access rights and attribute them to different kubernetes artifacts. For example:
1. Create a role  R that can only read certain secrets (database secrets)
2. Create a bunch of Api services and assign the role R to them
3. If ever another pod is compromised/hacked/broken then it cannot access the database secrets because it does not have the correct role for it

Here is a longer (better) article on this topic: https://medium.com/containerum/configuring-permissions-in-kubernetes-with-rbac-a456a9717d5d

File: 02-management-dashboard/README.md [hunk 4]
```diff
@@ -1,37 +1,58 @@
...
+
+For more info on what the Kubernetes dashboard is and how to run it check out the [github repository](https://github.com/kubernetes/dashboard).
 
 # Setting up the dashboard
 
 ## Start dashboard
 
-Create dashboard:
-```
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
```
[the remark this answers is not in the document]
Comment (mostaphaRoudsari) [reply 02-management-dashboard/README.md]: That sounds good to me but then why did you copied the other yaml locally? Should we keep both?

File: 02-management-dashboard/README.md (hunk 1 above)
Comment (mostaphaRoudsari) [reply 02-management-dashboard/README.md]: Indeed!

File: 02-management-dashboard/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (AntoineDao) [reply 02-management-dashboard/README.md]: Mostly as a convenience so that I could read it and understand how it works.

**Comment by mostaphaRoudsari** [comment]
@AntoineDao, I made adjustments based on your comments. Also updated the next 2 ReadMe files.

**Comment by AntoineDao** [comment]
I'll let you merge that one into master when you're ready.

**Pull request merged**